### PR TITLE
6.2.0

### PR DESCRIPTION
**Pull request by evanshortiss**
Addressing #121 #126 #112

### CODE REVIEW

**Comment by evanshortiss** [comment]
Hi @nirsky @moltar @satazor @Brigadier1-9 @DigiPie, this release has a decent few changes so I'd be grateful if you could try `npm install env-var@6.2.0-alpha.0` and let me know if you run into any issues :sweat_smile:

File: README.md [hunk 1]
```diff
@@ -405,10 +475,29 @@ const commaArray = env.get('DASH_ARRAY').asArray('-');
 const enumVal = env.get('ENVIRONMENT').asEnum(['dev', 'test', 'live'])
 ```
 
+### accessors
+A property that exposes the built-in accessors that this module uses to parse
+and validate values. These work similarly to the *asString()* and other
+accessors exposed on the *variable* type documented below, however they accept
```
Comment (DigiPie) [inline README.md:481]: Do you mean this instead?

```suggestion
accessors exposed on the *variable* type documented above, however they accept
```

File: README.md [hunk 2]
```diff
@@ -405,10 +475,29 @@ const commaArray = env.get('DASH_ARRAY').asArray('-');
...
+```js
+const env = require('env-var')
+
+// Validate that the string is JSON, and return the parsed result
+const myJsonDirectAccessor = env.accessors.asJson(process.env.SOME_JSON)
+
+const myJsonViaEnvVar = env.get('SOME_JSON').asJson()
+```
+
+All of the documented *asX()* accessors below are available. These are useful
```
Comment (DigiPie) [inline README.md:493]:
```suggestion
All of the documented *asX()* accessors above are available. These are useful
```

File: README.md [hunk 3]
```diff
@@ -113,6 +113,55 @@ const env = require('env-var')
...
+To enable logging you need to create an `env-var` instance using the `from()`
+function that the API provides and pass it a logger. A built-in logger is
+available, but a custom logger is also supported.
+
+Always exercise caution when logging environment variables!
+
+### Using the Built-in Logger
+
+The built-in logger will print logs unless `NODE_ENV` is set to either `prod`
+or `production`..
```
Comment (DigiPie) [inline README.md:130]:
```suggestion
or `production`.
```

**Comment by moltar** [comment]
All tests pass in my use cases. Thanks!

**Comment by satazor** [comment]
@evanshortiss I tested the alpha version and it's working as expected! Looking forward for the release.

**Pull request merged**